Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,288 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Text;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Analyzer 'SonarAnalyzer.Rules.CSharp.ObjectsShouldNotBeDisposedMoreThanOnce' threw an exception of type 'SonarAnalyzer.SymbolicExecution.SymbolicExecutionException' with message 'Error processing method: FindRichestPerson ## Method file: BalanceStats.cs ## Method line: 56,8 ## Inner exception: System.InvalidOperationException: This operation does not apply to an empty instance. ##   at System.Collections.Immutable.ImmutableStack`1[T].Peek () [0x0000d] in <2e5af6ef122c4468962237668e6f8259>:0  ##   at System.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5156603)